Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14.2 AS build-fe
+WORKDIR /src/
+COPY ui  .
+RUN yarn
+RUN yarn build
+
+FROM golang:1.14-alpine AS build-be
+WORKDIR /src/
+COPY . .
+RUN go build ./cmd/...
+
+
+FROM alpine:latest
```

Comment:
This could actually be `FROM scratch`, we don't need a base system here (if we do a build with CGo disabled)